guides/common/modules/ref_ports-and-firewall-requirements.adoc

```diff
@@ -35,9 +35,10 @@ ifdef::katello,satellite,orcharhino[]
 | 443, 80 | TCP | HTTPS, HTTP | Client | Content Retrieval | Content
 | 443, 80 | TCP | HTTPS, HTTP | {SmartProxy} | Content Retrieval | Content
 | 443, 80 | TCP | HTTPS, HTTP| Client | Content Host Registration | {SmartProxy} CA RPM installation
-| 443 | TCP | HTTPS | {ProjectName} |Content Mirroring | Management
+| 443 | TCP | HTTPS | {ProjectName} | Content Mirroring | Management
 | 443 | TCP | HTTPS | {ProjectName} | {SmartProxy} API | Smart Proxy functionality
-| 5646 | TCP | AMQP |{SmartProxy}| Katello agent | Forward message to Qpid dispatch router on {Project} (optional)
+| 1883 | TCP | MQTT | Client | Pull based REX (optional) | Content hosts for REX job notification (Optional)
```

Comment:
"optional" in the last column.

Reply:
Oops Sorry. I should have gone through it once. 😬